configs/tslint.json

```diff
@@ -1,6 +1,10 @@
 {
   "defaultSeverity": "error",
   "rules": {
+    "file-header": [
+      true,
+      "Copyright \\d{4}"
```

Comment:
@lSindra thanks for looking into it, unfortunately, such pattern does not match our header

`"\n \\* Copyright \\(C\\) \\d{4} .*\n"` should work better